synapse/models/orgs.py

```diff
@@ -43,7 +43,7 @@ def getBaseModels():
                 ('ou:hashost', {'subof': 'comp', 'fields': 'org,ou:org|host,it:host'}),
                 ('ou:hasemail', {'subof': 'comp', 'fields': 'org,ou:org|email,inet:email'}),
                 ('ou:hasphone', {'subof': 'comp', 'fields': 'org,ou:org|phone,tel:phone'}),
-                ('ou:hasnetuser', {'subof': 'comp', 'fields': 'org,ou:org|netuser,inet:netuser'}),
```

Comment:
Nitpick, but while we're updating is it worth changing the terminology across the board - that is, from ou:hasnetuser to ou:haswebacct (since we're changing netuser -> web:acct)?

Reply:
Nice catch. I think we probably should. @invisig0th ?

Reply:
fixed